tee/prototype/epoch.go

```diff
@@ -3,81 +3,198 @@
 package prototype
 
 import (
+	"errors"
+	"fmt"
 	"math/big"
 
 	"github.com/ethereum/go-ethereum/common"
-	log "github.com/sirupsen/logrus"
-
 	"github.com/perun-network/erdstall/tee"
+	log "github.com/sirupsen/logrus"
 )
 
-// An Epoch stores all balances of an epoch and contains signalling
-// infrastructure.
 type (
+	// Epoch manages deposits, transactions, and exits, as well as the
+	// progression of time in the system.
 	Epoch struct {
-		Number tee.Epoch
+		number tee.Epoch // Current deposit epoch.
+
+		exitLocked map[common.Address]struct{} // Withdrawing at end of epoch.
+		exitReqs   map[common.Address]struct{} // Requested exits.
+		balances   map[common.Address]*Acc     // Latest balances.
```

Comment:
IMO the field name should be consistent with the type, so
```suggestion
		accs   map[common.Address]*Acc     // Latest account states.
```
or `accounts` or
```suggestion
		balances   map[common.Address]*Balance     // Latest balances.
```

Reply:
accs